### PR TITLE
WIP: Add URL state to search

### PR DESCRIPTION
**Pull request by unclecheese**
https://github.com/silverstripe/silverstripe-admin/issues/938

https://github.com/silverstripe/silverstripe-admin/issues/814

### CODE REVIEW

File: client/src/components/Search/Search.js [hunk 1]
```diff
@@ -362,8 +377,9 @@ class Search extends Component {
 
     const searchText = searchData[name] || '';
     // Data to store in the redux state
+    console.log('overrides are ', overrides);
```
Comment (sminnee) [inline client/src/components/Search/Search.js]: remember to remove these before merging.

Comment (unclecheese) [reply client/src/components/Search/Search.js]: Yeah, the build won't allow it anyway. Long way from merge. :-)

**Comment by emteknetnz** [comment]
This pull request hasn't had any activity for a while. Are you going to be doing further work on it, or would you prefer to close it now?

**Comment by emteknetnz** [comment]
It seems like there's going to be no further activity on this pull request, so we've closed it for now. Please open a new pull-request if you want to re-approach this work, or for anything else you think could be fixed or improved.

**Pull request closed without merging**